src/js/components/Menu/stories/Justify.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import { Box, Grommet, Menu } from 'grommet';
+import { User } from 'grommet-icons';
+import { hpe } from 'grommet-theme-hpe';
+
+export const Justify = () => (
+  <Grommet theme={hpe}>
+    <Box pad="small">
+      <Menu
+        alignSelf="start"
+        label="Actions"
+        items={[
+          { label: 'Hey', icon: <User />, justify: 'center' },
+          { label: 'Hello', icon: <User />, justify: 'end' },
+          { label: 'Hii', icon: <User /> },
+        ]}
+      />
+    </Box>
+  </Grommet>
+);
+
```

Comment:
I think we should disable this story from chromatic snapshot tests since the behavior we want to test is happening when you click on the menu button.
```suggestion
Justify.parameters = {
  chromatic: { disable: true },
};
```